lib/go-tc/tovalidate/rules.go

```diff
@@ -46,3 +48,19 @@ func IsOneOfStringICase(set ...string) func(string) bool {
 	}
 	return IsOneOfString(lowcased...)
 }
+
+func IsGreaterThanZero(value interface{}) error {
```

Comment:
This function could be refactored to here: https://github.com/apache/trafficcontrol/blob/master/lib/go-tc/tovalidate/rules.go#L19

Reply:
am confused, this function _is_ there

Reply:
sorry I didn't notice the package changed from above, ignore my previous comment